front/src/styles/scss/_variables.scss

```
@@ -32,6 +32,18 @@ $colors: (
   'coolgray5': #b9b9b9,
   'coolgray3': #d7d7d7,
   'coolgray1': #f2f2f2,
+  'grey60': #5c5955,
+  'ambiantB5': #faf9f5,
+  'ambiantB10': #f7f6ee,
+  'ambiantB15': #f2f0e4,
+  'selection20': #fff2b3,
+  'warning30': #eaa72b,
+  'warning60': #7d521e,
+  'error60': #d91c1c,
+  'error80': #6b0000,
+  'error5': #ffeeed,
+  'primary60': #1844ef,
+  'primary80': #1f0f96,
```

Maybe we could sort the variables and separate them from the "old" colors

```suggestion

  // OSRD Design System
  'grey60': #5c5955,
  'ambiantB5': #faf9f5,
  'ambiantB10': #f7f6ee,
  'ambiantB15': #f2f0e4,
  'selection20': #fff2b3,
  'warning30': #eaa72b,
  'warning60': #7d521e,
  'error60': #d91c1c,
  'error80': #6b0000,
  'error5': #ffeeed,
  'primary60': #1844ef,
  'primary80': #1f0f96,
```

---

You're right, it's done